torchbox/wagtail-torchbox
# Adds the BE for the new courses app...

This PR is the first part of the courses work. 
Designs: https://www.figma.com/file/H5pQECphbbYsvSWtag3OrCWC/Torchbox?type=design&node-id=3309-9169&mode=design&t=TxEdRwPd42H9EKdS-4

Note: This is part of a short 2 day brief that does not have any tickets created. FE will be done by Simon based of this.

### Description of Changes Made

In this PR, the following is added:
- New course detail page
- 2 new block types for the course detail page
- Stubbs out the course landing page model

**What's not been done:**
- The pattern library is currently broken and not rendering any pages. I have not fixed (or caused) this as part of this work

### How to Test

Checkout this branch locally.
Add a course landing page
Add a child course detail page.
You should be able to add everything needed for the course detail page, including the 2 new blocks.


### MR Checklist
